tcms/testplans/templates/testplans/get.html

```diff
@@ -110,26 +112,51 @@ <h2 class="card-pf-title">
 <template id="test_case_row">
     <div class="list-group-item">
         <div class="list-group-item-header">
-            <div class="list-view-pf-expand">
-                <span class="fa fa-angle-right"></span>
-            </div>
-            <div class="list-view-pf-checkbox">
-                <input type="checkbox">
-            </div>
-            <div class="list-view-pf-actions">
-                <div class="dropdown pull-right dropdown-kebab-pf">
-                    <button class="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight9"
-                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
-                        <span class="fa fa-ellipsis-v"></span>
-                    </button>
-                    <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebabRight9">
-                        <li><a href="#">{% trans 'Change status' %}</a></li>
-                    </ul>
-                </div>
-            </div>
             <div class="list-view-pf-main-info">
-                <!-- todo: this text is slightly misaligned. for some reason goes to the top -->
-                <a class="testcase" href="#"></a>
+                <div class="list-view-pf-left">
+                    <span class="fa fa-angle-right"></span>
+                </div>
+                <div class="list-view-pf-checkbox">
```

Comment:
Add `style="margin-top: 0; margin-bottom: 0"` - will cause the entire row to be smaller in height, matching what we have in the TC page. The right border for visual styling around this checkbox makes the rows higher than usual.